tokenLists/terra2.tokenlist.json

```diff
@@ -246,6 +246,20 @@
       "token": "terra16q6kqp0p2hlfgj52sadv5p7qh2hgkkjdfske48vklz94tkxg4jwq8k8myj",
       "icon": "../img/SAYVE/Sayve.svg",
       "decimals": 6
+    },
```

Comment:
can you please move those tokens under "phoenix-1"? they're currently under "pisco-1" which is not correct considering those contracts are on terra mainnet. thanks!

Reply:
I moved everything under phoenix-1, sorry for the inconvenience

Reply:
thanks!